sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -573,4 +573,48 @@ object DateTimeUtils {
       dayInYear - 334
     }
   }
+
+  /**
+   * Returns Day of week from String. Starting from friday, marked as 0.
+   */
+  def getDayOfWeekFromString(string: UTF8String): Int = {
```

Comment:
Can you document inline why we choose Fri as 0, since it is a weird choice (usually Sun or Mon is considered the beginning of the week). Is it because 1970-1-1 is a Thu?